galois_reed_muller.yml

```diff
@@ -0,0 +1,40 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# code id, physical, logical are all lower case
+code_id: galois_reed_muller
+physical: galois
+logical: galois
+
+name: 'Nonbinary Quantum Reed-Muller Codes'
+introduced: '\cite{doi:10.1109/18.771249}'
+
+description: 'The pure stabilizer code \([[q^m,k(v_2)-k(v_1),min{d(v_1^{\perp}),d(v_2)}]]_{q}\) with parameters \(k(v) = \sum_{j=0}^{m}(-1)^{j}\dbinom{m}{j}\dbinom{m+v-jq}{v-jq}\) and 
+\(d(v) = (R+1)q^{Q}\), where \(m(q-1)-v=(q-1)Q+R\) so that \(0\leq R\leq\ q-1) is called quantum Reed Muller code. Here \(0\leq v_1,v_2 leq m(q-1)-1\), q is prime and m is positive integer.'
+
+features:
+
+  rate: '\(k(v) = \sum_{j=0}^{m}(-1)^{j}\dbinom{m}{j}\dbinom{m+v-jq}{v-jq}\) and \(n = q^{m}\) so rate must be \(\frac{\sum_{j=0}^{m}(-1)^{j}\dbinom{m}{j}\dbinom{m+v-jq}{v-jq}}{q^m}\).'
+  
+  encoder: 'A linear code \([n,k]_{q^{2}}\) contained in its hermitian dual implies the existence of \([[n,n-2k,d]]_{q}\) quantum code where \(d = min(wt(C^{\perp_{h}}\C))\). Taking C
+  as \(\mathcal{R}_{q^2}(v,m)\) for \(0\leq v \leq q(m-1)-1\) yields a pure \([[q^{2m},q^{2m}-k(v),d(v^{\perp}]]_{q}\) quantum code where 
+  \(k(v) = \sum_{j=0}^{m}(-1)^{j}\dbinom{m}{j}\dbinom{m+v-jq^2}{v-jq^2}\) and \(d(v^{\perp}) = (R+1)q^{2Q}\) with \(v+1 = (q^2 - 1)Q + R\).'
+  
+  protection: '\(\mathcal{Q}\mathcal{R}\mathcal{M}_{d}(m)\) quantum codes are \(\mathcal{M}_{d}^{m}\) distillation codes of distance D=2.'
```

Comment:
Define distillation code. Do we need a stub entry for it? Is it a cousin/parent?